packages/hub/src/relative-values/components/views/PlotView/ForcePlot.tsx

```diff
@@ -173,7 +173,7 @@ export const ForcePlot: FC<{
     simulation.nodes(nodes);
 
     return simulation;
-  }, [model, nodes]);
+  }, [model, nodes, forceFn]);
```

Comment:
This lint warning was bothering me for a while (it's the only non-trivial lint warning we have in the entire repo, I think...), but this is not the right fix.

`forceFn` identity is unstable, and this dependency would lead to `simulation` object being rebuilt every time, which will trigger simulation restarts on each rerender.

This is probably why https://quri-hub-git-relative-value-fixes-quantified-uncertainty.vercel.app/models/ozzie/testt/relative-values/fn/plot flickers when I move the cursor over the plot.

IIRC stabilizing `forceFn` identity is hard; we could lie about it with refs, or just eslint-ignore this line.

Reply:
Easy to fix, thanks!